config/serialization/CustomerType.yaml

```
@@ -0,0 +1,6 @@
+App\Customer\Domain\Entity\CustomerType:
+  attributes:
+    id:
+      groups: ['read:customerType']
+    value:
+      groups: ['read:customerType', 'write:customerType']
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)